spk/duplicity/src/installer.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+# Package
+PACKAGE="duplicity"
+DNAME="Duplicity"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python"
+VIRTUALENV="${PYTHON_DIR}/bin/virtualenv"
+PATH="${INSTALL_DIR}/env/bin:${INSTALL_DIR}/bin:${PYTHON_DIR}/bin:${PATH}"
+
+preinst ()
+{
+    exit 0
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+
+    # Create a Python virtualenv
+    ${VIRTUALENV} --system-site-packages ${INSTALL_DIR}/env > /dev/null
```

Comment:
I think you can get away with removing `--system-site-packages`. In doing that, the package is going to be easier to maintain in the long run. Could you check if that works?